boards/arm/arduino_nicla_sense_me/arduino_nicla_sense_me_defconfig

```diff
@@ -0,0 +1,25 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52832_QFAA=y
+CONFIG_BOARD_ARDUINO_NICLA_SENSE_ME=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable sam-ba bootloader
+CONFIG_BOOTLOADER_BOSSA=y
```

Comment:
board.cmake is only declaring support for pyocd, so why is this enabling bossa here? I would have expected support for the bossac runner enabled in board.cmake.

Reply:
To my understanding bossa doesn't support SAMD11 so I'll just remove this line 👍